### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by manifestinteractive**
Looking to get at least one set of eyes of this for peer review before merging it into master.  Don't need to actually install the extension ( though that would be a bonus ), as I have a few people testing this already.  

Mainly just lookin for:

1. Code Quality Review
2. Make sure documentation seems reasonably clear and free from obvious errors, and Brand specific info is on point
3. Make sure screenshots do not contain any client IP ( I made an extended effort to anonymize path names, etc )

### CODE REVIEW

File: .firefox/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "manifest_version": 2,
+  "name": "SFCC DevTools",
+  "version": "1.0.0",
+  "description": "Support Browser Interaction with VS Code, Eclipse, and SFCC Business Manager via DevTools Web Inspector.",
+  "author": "Red van Workshop",
```
Comment (sergeystoma) [inline .firefox/manifest.json]: A totally Dutch vibe, but not quite a proper name

Comment (manifestinteractive) [reply .firefox/manifest.json]: Fixed :)

File: src/sidebar.js [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+          browser.devtools.inspectedWindow.eval(`{ window.location.href }`).then(getPageInfo, onError)
+        }
+      } else {
+        $sidebar.innerHTML = '<i>No SFCC Comment Selected</i>'
+      }
+    }
+
+    // Use inspector to parse DOM and return node if Comment Block ( nodeType === 8 )
+    if (browser.devtools) {
+      browser.devtools.inspectedWindow.eval(`{ const node = $0; node.nodeType === 8 && node.nodeValue; }`).then(selectedElement, onError)
```
Comment (johnkramlich) [inline src/sidebar.js]: The node type constant may be easier to understand
```suggestion
      browser.devtools.inspectedWindow.eval(`{ const node = $0; node.nodeType === Node.COMMENT_NODE && node.nodeValue; }`).then(selectedElement, onError)
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "sfcc-devtools",
+  "version": "1.0.0",
+  "description": "Support Browser Interaction with VS Code, Eclipse, and SFCC Business Manager via DevTools Web Inspector.",
+  "author": "Red van Workshop",
+  "license": "MIT",
+  "private": false,
```
Comment (johnkramlich) [inline package.json:7]: Consider adding a repository property
```suggestion
  "private": false,
  "repository": "github:redvanworkshop/sfcc-devtools",
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "sfcc-devtools",
+  "version": "1.0.0",
+  "description": "Support Browser Interaction with VS Code, Eclipse, and SFCC Business Manager via DevTools Web Inspector.",
+  "author": "Red van Workshop",
+  "license": "MIT",
```
Comment (johnkramlich) [inline package.json:6]: Consider adding an engines property to specify a minimum node version.  I was not able to get npm test to work when my node version was v6.14.4.  It did work successfully on v10.15.0.

https://flaviocopes.com/package-json/#engines

Comment (manifestinteractive) [reply package.json:6]: Good call, I will copy over what we are using in our other repos

File: package.json (hunk 3 above)
Comment (manifestinteractive) [reply package.json:7]: Good call, I will copy over what we are using in our other repos

**Pull request merged**